### ISSUE TITLE
for Windows installation??

### ISSUE THREAD

**Issue by ykseiljon** [GeoffMontee/tds_fdw#32]
Hello, is any version for Windows?

**Comment by GeoffMontee** [comment]
Hi @ykseiljon,

I haven't tried compiling tds_fdw on Windows, but it might work. FreeTDS can supposedly be built on Windows:

http://www.freetds.org/userguide/osissues.htm#WINDOWS

**Comment by ykseiljon** [comment]
Thanks for answer, i'll try. From Fedora works very well.

**Comment by win32nipuh** [comment]
Hi, did you try to compile and install it in Windows version?

**Comment by ykseiljon** [comment]
No, I didn't, this week I will do it.

**Comment by ivomts** [comment]
ykseiljon, did you already try it on windows?

**Comment by jaitor1** [comment]
@ykseiljon @ivomts @win32nipuh @GeoffMontee @juliogonzalez Did anyone succeed compiling tds_fdw for Windows??

**Comment by ivomts** [comment]
No. I didn't even tried it yet... :(

I've temporarily overcome my needs with a small python script.



On 5 April 2016 at 10:13, jaitor1 <notifications@github.com> wrote:

> @ykseiljon <https://github.com/ykseiljon> @ivomts
> <https://github.com/ivomts> @win32nipuh <https://github.com/win32nipuh>
> @GeoffMontee <https://github.com/GeoffMontee> @juliogonzalez
> <https://github.com/juliogonzalez> Did anyone succeed compiling tds_fdw
> for Windows??
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GeoffMontee/tds_fdw/issues/32#issuecomment-205722396>
>



-- 
Ivo Silvestre
______________________
*técnico sig | gis technician*

<https://www.linkedin.com/in/ivomts>
______________________

**Comment by GeoffMontee** [comment]
I found out today that tds_fdw 1.0.7 is distributed with [BigSQL](http://www.bigsql.org/), which is available for Windows. You can see it listed on their [components page](http://www.bigsql.org/se/components.jsp).

It might be possible to use their tds_fdw shared library with a regular PostgreSQL installation on Windows.

**Comment by jaitor1** [comment]
Great news @GeoffMontee I'll try to make it next week!

**Comment by eddydev03** [comment]
Almost 4 months now but anyway. @jaitor1 Did you succeed to use tds_fdw with the BigSQL distrib? :)
Still does not work for me :(. If you found another way to get data from a remote mssql, I will appreciate tips :).

**Comment by jaitor1** [comment]
@eddydev03 No I didn't.... :(

**Comment by eddydev03** [comment]
@jaitor1 Yep, not that easy to use an fdw between posgres and mssql on Windows. Thanks for the quick answer by the way :)

**Comment by ykseiljon** [comment]
I tried, but not worked. Sorry to be later (I was off-line for a long time unexpectedly ). I'll try it again.

**Comment by jaitor1** [comment]
Anyone made it work in Windows?
